frontend/src/components/CodeBrowser/CodeLine/index.tsx

```diff
@@ -93,18 +100,49 @@ const CodeLine: React.FC<ICodeLine> = ({ path, line, isDiff, code }) => {
     const data = new FormData(form);
     const fb: IGraderFeedback = {
       path,
-      line: Number(data.get("line")),
+      line,
       body: String(data.get("comment")).trim(),
       points: Number(data.get("points")),
       ta_username: currentUser.login,
     };
     if (fb.points == 0 && fb.body == "") return;
     if (fb.body == "") fb.body = "No comment left for this point adjustment.";
-    addFeedback(fb);
+    addFeedback([fb]);
     setEditing(false);
     form.reset();
   };
 
+  const attachRubricItems = (riIDs: number[]) => {
+    if (
+      !currentUser ||
+      !selectedClassroom ||
+      !assignmentID ||
+      !studentWorkID ||
+      !rubric
+    )
+      return;
```

Comment:
Should there be an error state to inform users why this didn't work? Same goes elsewhere.

Reply:
this is just a typescript async thing since technically certain values are typed as x | null or x | undefined, but to access this page they are guaranteed to have some value